src/ansys/geometry/core/sketch/sketch.py

```diff
@@ -485,6 +485,88 @@ def arc_from_three_points(
         arc = Arc.from_three_points(start, inter, end)
         return self.edge(arc, tag)
 
+    def arc_from_start_end_and_radius(
+        self,
+        start: Point2D,
+        end: Point2D,
+        radius: Union[Quantity, Distance, Real],
+        convex_arc: Optional[bool] = False,
+        clockwise: Optional[bool] = False,
+        tag: Optional[str] = None,
+    ) -> "Sketch":
+        """
+        Add an arc to the sketch plane from the start and end points and a given radius.
+
+        Parameters
+        ----------
+        start : Point2D
+            Starting point of the arc.
+        end : Point2D
+            Ending point of the arc.
+        radius : Union[~pint.Quantity, Distance, Real]
+            Radius of the arc.
+        convex_arc : bool, default: False
+            Whether the arc is convex. The default is ``False``.
```

Comment:
This description isn't consistent with the earlier ones--where the behavior for both options is described.

Reply:
Let me adapt. Thanks!